proto/tabletmanagerdata.proto

```diff
@@ -160,6 +160,14 @@ message GetPermissionsResponse {
   Permissions permissions = 1;
 }
 
+message GetServerStatusRequest {
+  repeated string statuses = 1;
```

Comment:
Isn't it better to call this `status_variables`? or even `global_status_variables`?

Reply:
I have changed the function name as Matt suggested and this now seems a little overkill.

Reply:
With that change, it's probably more intuitive to call these `variables` instead of `statuses`, but I won't insist on it.

Reply:
Changed to `variables`